packages/element-lib/src/sidetree/getTransactions.js

```diff
@@ -23,12 +23,18 @@ module.exports = (sidetree) => {
         end = start + count;
       }
       transactions = await sidetree.blockchain.getTransactions(start, end, { omitTimestamp });
-      // Update the cache
-      const cachedTransactionsPromises = transactions.map(transaction => sidetree.db.write(`element:sidetree:transaction:${transaction.transactionTimeHash}`, {
-        type: 'element:sidetree:transaction',
-        ...transaction,
-      }));
-      await Promise.all(cachedTransactionsPromises);
+
+      // this is very likely to fail always after the first run...
```

Comment:
Mmmmh, does wrapping this in a try catch and ignoring the error solve this problem?

Reply:
Sortof... the problem is that we don't have a clean way of knowing: is this transaction ranged already cached... I'm gonna open a new ticket for this: https://github.com/decentralized-identity/element/issues/89